domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -427,52 +487,52 @@ class TopicListControllerTest {
     )
     testCoroutineDispatchers.runCurrent()
 
-    topicListController.getOngoingStoryList(profileId0).toLiveData()
-      .observeForever(mockOngoingStoryListObserver)
+    topicListController.getRecommendedActivityList(profileId0).toLiveData()
+      .observeForever(mockRecommendedActivityListObserver)
     testCoroutineDispatchers.runCurrent()
 
-    verifyGetOngoingStoryListSucceeded()
+    verifyGetRecommendedActivityListSucceeded()
 
-    val ongoingTopicList = ongoingStoryListResultCaptor.value.getOrThrow()
-    assertThat(ongoingTopicList.recentStoryCount).isEqualTo(2)
-    verifyOngoingStoryAsRatioStory0Exploration0(ongoingTopicList.recentStoryList[0])
-    verifyOngoingStoryAsRatioStory1Exploration2(ongoingTopicList.recentStoryList[1])
```

Comment:
`verifyOngoingStoryAsRatioStory1Exploration2` and `verifyDefaultRecommendedActivityListSucceeded` this seems like an unused function now. We can remove the function definition.

Reply:
Done